### PR TITLE
Add chainalysis

### PR DESCRIPTION
**Pull request by juliangruber**

### CODE REVIEW

**Comment by juliangruber** [comment]
@bajtos hindsight please

File: index.js [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
-  res.statusCode = forbiddenAddresses.has(address) ? 403 : 200
+  const fetchRes = await fetch(
+    `https://public.chainalysis.com/api/v1/address/${address}`,
+    {
+      headers: {
+        'X-API-Key': apiKey,
+        accept: 'application/json'
+      }
+    }
+  )
```
Comment (bajtos) [inline index.js:14]: What happens if the Chainanalysis API is temporarily down? Should we add `p-retry` with e.g. 5 attempts?

**Review by bajtos — COMMENTED** [review]
LGTM 👍🏻

File: index.js (hunk 1 above)
Comment (juliangruber) [reply index.js:14]: If it's down the service responds with a 500, and the client can retry. I propose to make this more solid in the future (given timelines)

Comment (bajtos) [reply index.js:14]: 👌🏻

**Pull request merged**